### ISSUE TITLE
Error in execution

### ISSUE THREAD

**Issue by tmctmc123** [sheffield-bioinformatics-core/periscope#14]
Hello,
I have came across with the following message when I attempted to run periscope.

" File "/home/anaconda3/envs/periscope/lib/python3.6/site-packages/periscope/scripts/search_for_sgRNA_ont.py", line 9, in <module>  from artic.align_trim import find_primer

ModuleNotFoundError: No module named 'artic' "

Then the programme failed. I would like to ask if you know any possible solution to this issue?

**Comment by mattdmem** [comment]
Hello,

It looks like it can't find the artic package - did you manage to install the environment as explained in the readme?

```
git clone https://github.com/sheffield-bioinformatics-core/periscope.git && cd periscope
conda env create -f environment.yml
conda activate periscope
pip install .
```

If you did can you try this and let me know what happens?

```
conda activate periscope
artic -h
```

Matt

**Comment by tmctmc123** [comment]
Hello Matt,

Yes I did installed the periscope environment and ran the command with the activated environment. The following error message also popped up after "ModuleNotFoundError".

"Error in rule periscope:
    jobid: 3
    output: /home/test_periscope.bam, /home/test_periscope_counts.csv, /home/test_periscope_amplicons.csv, /home/test_periscope_novel_counts.csv
    shell:
        python /home/anaconda3/envs/periscope/lib/python3.6/site-packages/periscope/scripts/search_for_sgRNA_ont.py --bam /home/test.bam --score-cutoff 50 --output-prefix /home/test --score-cutoff 50 --sample barcode08_518178 --orf-bed /home/periscope/periscope/resources/orf_start.bed --primer-bed /home/periscope/periscope/resources/artic_primers_V1.bed --amplicon-bed /home/periscope/periscope/resources/artic_amplicons_V1.bed --tmp /tmp
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message"

**Comment by tmctmc123** [comment]
Also when I tried 'artic -h', it said 'artic: command not found'.

**Comment by llk578496** [comment]
we also got the exact same issue. We have installed artic through source and we still got the same error message. Thanks.

**Comment by mattdmem** [comment]
Artic is not installing properly, I will fix this today - apologies y'all!

**Comment by mattdmem** [comment]
Fixed (I hope) in commit `9be3db0` - I've released a new version - 0.0.10
